test/e2e/network_segmentation.go

```diff
@@ -354,19 +354,24 @@ var _ = Describe("Network Segmentation", func() {
 							if podIP.IP.To4() == nil {
 								ipCommand = append(ipCommand, "-6")
 							}
-							// 1. Find current default route and delete it
-							defRoute, err := e2ekubectl.RunKubectl(udnPod.Namespace,
+							// 1. Find current default routes and delete them
+							defRoutes, err := e2ekubectl.RunKubectl(udnPod.Namespace,
 								append(ipCommand, "route", "show", "default")...)
 							Expect(err).NotTo(HaveOccurred())
-							defRoute = strings.TrimSpace(defRoute)
-							if defRoute == "" {
+							if defRoutes == "" {
 								continue
 							}
-							framework.Logf("Found default route %v, deleting", defRoute)
-							cmd := append(ipCommand, "route", "del")
-							_, err = e2ekubectl.RunKubectl(udnPod.Namespace,
-								append(cmd, strings.Split(defRoute, " ")...)...)
-							Expect(err).NotTo(HaveOccurred())
+
+							// Remove last new line to propertly split them using new line
+							defRoutes = strings.TrimSuffix(defRoutes, "\n")
+
+							framework.Logf("Found default routes %v, deleting", defRoutes)
+							for _ = range strings.Split(defRoutes, "\n") {
+								cmd := append(ipCommand, "route", "del")
+								_, err = e2ekubectl.RunKubectl(udnPod.Namespace,
+									append(cmd, "default")...)
```

Comment:
> OK, so IIUC, here we just delete the N default routes on the pod; I was confused since you aren't using the route data to delete it.
> 
> While it's exactly the same, maybe it's more readable ?... Not a must, it's pretty much opinionated.

It simplify the test since we don't have to retrieve the route we just care for number of routes.

> 
> Finally, maybe encapsulating this in an helper would be good.

I don't want to refactor this test since it's not new, I am just fixing it.